### PR TITLE
Restructuring Rogue's Directory Structure - Documentation 

### PR DESCRIPTION
**Pull request by chloealee**
#### What's this PR do?
- We did the majority of directory restructuring in #654 but I forgot to restructure the docs! This PR does that. 

#### How should this be reviewed?
👀 

#### Relevant tickets
Finishes work in #654 

#### How to deploy
https://github.com/DoSomething/rogue/blob/master/docs/development/deployments.md

### CODE REVIEW

File: docs/endpoints/Legacy/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Rogue API
+This is the Rogue API, it is used to capture activity from members.
+
+Most recent stable release is V3
+
+### Documentation
+[API v3](/docs/endpoints/v3/README.md)
+
+<!-- ## Endpoints
```
Comment (DFurnes) [inline docs/endpoints/Legacy/README.md]: Should all these endpoints be commented out? 👀

File: SUMMARY.md [hunk 2]
```diff
@@ -13,14 +13,13 @@
...
-- [Web API (depricated)](/docs/endpoints/web/README.md)
-- [API v1 (depricated)](/docs/api/v1/README.md)
-- [API v2 (depricated)](/docs/api/v2/README.md)
-- [API v3](/docs/endpoints/v3/README.md)
-  - [Signups](docs/endpoints/v3/signups.md)
-  - [Posts](docs/endpoints/v3/posts.md)
-  - [Reviews](docs/endpoints/v3/reviews.md)
-  - [Tags](docs/endpoints/v3/tags.md)
-
+- [Web API (depricated)](/docs/endpoints/Legacy/web/README.md)
```
Comment (DFurnes) [inline SUMMARY.md]: depricated → deprecated

Comment (chloealee) [reply SUMMARY.md]: oops!

File: docs/endpoints/Legacy/README.md (hunk 1 above)
Comment (chloealee) [reply docs/endpoints/Legacy/README.md]: you're right they shouldn't be!

**Pull request merged**